reproducibility_project/spe_subproject/README.md

```diff
@@ -0,0 +1,96 @@
+## You are here
+Single Point Energy (SPE) subproject for reproducibility study
+
+## How to set up single energy calculations
+Single point energies are calculated from the snapshots found in `src/system_snapshots`. These are created from the file `src/create_snapshots.py`. This utilizes the system builder in the main study to create initial configurations that can be compared across simulation engine for energy calculations.
+
+**NOTE: Packmol is used to pack the simulations box, and can generate different packings with the same seed across Linux and macOS operating distributions. The simulation snapshots used in this study were created using macOS Catalina Version 10.15.7.**
+
+The energy calculations will all be output to `log-spe.txt` and located in the job workspace directories. All energy units will be reported in **extensive**(no division by number of moleules) energies of kJ/mol.
+
+The energy will be logged as space separated columns with the first row containing headers. They will be as follows:
+total_energy potential_energy kinetic_energy vdw_energy coul_energy pair_energy bonds_energy angles_energy dihedrals_energy
+
+The following energy definitions are calculated using the [lammps thermo](https://docs.lammps.org/thermo_style.html) calculation methods.
+total_energy = etotal
+potential_energy = pe
+kinetic_energy = ke
+vdw_energy = evdwl
+coul_energy = ecoul
+pair_energy = epair
```

Comment:
What is the difference between vdw energy and pair energy?

Reply:
right now the energies I'm able to log in hoomd are:
LJ pair, Ewald, Coulomb, LJ pair (from the 1-4 interactions), Coulomb (from the 1-4 interactions), Harmonic bond, Harmonic angle, OPLS dihedral, total kinetic_energy, and total potential_energy

Reply:
vdw_energy is just the vanderwaal energy plus the long range tail energy.
pair_energy is the vanderwaal energy + coulombic energy + long-range kspace energy.

It's all documented in lammps thermo_style manual.

So it seems like in your case:
vdw_energy = LJ pair + LJ_pair (1-4) assuming your tail correction is accounted for.
pair_energy = vdw_energy + Coulomb + Coulomb (1-4) + Ewald